src/server/qtquick/private/wquickcoordmapper.cpp

```diff
@@ -31,7 +31,7 @@ class Q_DECL_HIDDEN WQuickCoordMapperPrivate : public QQuickItemPrivate
         return item->d_func();
     }
 
-    WQuickObserver *target;
+    QPointer<WQuickObserver> target;
```

Comment:
destroy WQuickCoordMapper in WQuickCoordMapperHelper::get if target is destroyed.

Reply:
done